### PR TITLE
Update area owners to include lead owners column

### PR DESCRIPTION
**Pull request by danmoseley**
The goal here is to have a single responsible individual for each area. It's not perfect because some areas have multiple mappings but it's good enough that we use this for reporting/accountability.

We have some gaps - would be good to fill in the rest of the leads

See rich diff: https://github.com/dotnet/runtime/pull/37760/files?short_path=09cda71#diff-09cda71b39848d77d475fecb2acac91a

### CODE REVIEW

**Comment by stephentoub** [comment]
> The goal here is to have a single responsible individual for each area.

What is the intent regarding who someone tags / contacts?  Presumably not the lead.

**Comment by danmosemsft** [comment]
> What is the intent regarding who someone tags / contacts? Presumably not the lead.

No. Hmm, one thing we could do to avoid that is add the names as `(1)` `(2)` etc and list them at the bottom, which would make their ID's less prominent in the table?

**Comment by danmosemsft** [comment]
cc @ericstj @jeffhandley  - for reporting we need one lead per area, I arbitrarily picked Jeff for now as owner of System.Runtime. We can change if needed..

**Comment by marek-safar** [comment]
@danmosemsft would it make sense to sort it alphabetically?

**Comment by danmosemsft** [comment]
Yup, sorted. It's now impossible to review easily. @jeffschwMSFT can you help me fill in lead owners for your areas? I could guess based on address book in many cases?

**Comment by jeffschwMSFT** [comment]
Sorry, I am late to the conversation.  It is often nice to have more than one person in this list - as the list is used for multiple purposes.  There is 'who can I ask' which is nice to have more than 1, and there is the 'who is accountable for' and that mapping has not been that hard to track.  How can I help with this effort?

**Comment by danmosemsft** [comment]
@jeffschwMSFT that's what we have -- a lead column, for the accountable person, and the (already existing) area owners column, for the engineers who maintain that area.

see: https://github.com/dotnet/runtime/blob/3cc66890f33d941a85eb5e88c8962b20bec02453/docs/area-owners.md

I would like to fill in more of the leads column.

My assumption is that this file remains the source of truth, and you can consume it in your reporting queries (I just regexified it in the past to make quoted comma separated strings to stick in the query - happy to do this again if it helps)

**Comment by jeffschwMSFT** [comment]
;) got it

**Review by marek-safar — APPROVED** [review]

File: docs/area-owners.md [hunk 1]
```diff
@@ -1,122 +1,120 @@
...
+| area-Extensions-Primitives                     | @ericstj      | @maryamariyan                                       |                                                                                                                                                                                                                                                                                           |
+| area-GC-coreclr                                | @jeffschw     | @Maoni0                                             |                                                                                                                                                                                                                                                                                           |
```
Comment (jeffschwMSFT) [inline docs/area-owners.md:30]: jeffschw => jeffschwmsft (and there are others)

File: docs/area-owners.md [hunk 2]
```diff
@@ -1,122 +1,120 @@
...
+| area-ReadyToRun-coreclr                        | @mangod9      | @nattress                                           |                                                                                                                                                                                                                                                                                           |
+| area-SDK                                       | @mangod9      | @janvorli                                           | General development issues and overlap with the SDK and CLI                                                                                                                                                                                                                               |
```
Comment (jeffschwMSFT) [inline docs/area-owners.md:50]: This is more of leecoward

File: docs/area-owners.md [hunk 3]
```diff
@@ -1,122 +1,120 @@
...
+| area-System.Runtime.CompilerServices           | @ericstj      | @Anipik @steveharter                                |                                                                                                                                                                                                                                                                                           |
+| area-System.Runtime.InteropServices            | @ericstj      | @AaronRobinsonMSFT @jkoritzinsky                    | Excluded:<ul><li>System.Runtime.InteropServices.RuntimeInfo</li></ul>                                                                                                                                                                                                                     |
```
Comment (jeffschwMSFT) [inline docs/area-owners.md:96]: the owner is jeffschwmsft

**Review by jeffschwMSFT — CHANGES_REQUESTED** [review]

**Comment by danmosemsft** [comment]
OK. Look good now? There's still a couple without leads, but I think that's enough to give us accountable reporting. 

@cheenamalhotra @David-Engel is there a lead we can put down for SqlClient (don't know Ken's ID), or one of you?

File: docs/area-owners.md (hunk 2 above)
Comment (jeffschwMSFT) [reply docs/area-owners.md]: Thanks, I should also note that janvorli is not the right sub owner either.  Lee hopefully would know who is.

**Comment by David-Engel** [comment]
@danmosemsft 

> is there a lead we can put down for SqlClient (don't know Ken's ID), or one of you?

You can put me on there.

**Comment by danmosemsft** [comment]
@jeffschwMSFT can you sign off (since you blocked merging)? feel free to merge

**Comment by jeffschwMSFT** [comment]
@danmosemsft there are still 2 outstanding tweaks needed.

File: docs/area-owners.md (hunk 2 above)
Comment (danmosemsft) [reply docs/area-owners.md]: Well, I put @leecow for now.

**Comment by danmosemsft** [comment]
Tweaked

**Review by jeffschwMSFT — APPROVED** [review]

**Pull request merged**